src/librustdoc/html/render/print_item.rs

```diff
@@ -1745,7 +1745,13 @@ fn item_variants(
 }
 
 fn item_macro(w: &mut Buffer, cx: &mut Context<'_>, it: &clean::Item, t: &clean::Macro) {
-    highlight::render_item_decl_with_highlighting(&t.source, w);
+    wrap_item(w, |w| {
+        // FIXME: Also print `#[doc(hidden)]` for `macro_rules!` if it `is_doc_hidden`.
```

Comment:
preexisting, will fix separately bc. there's another bug related to this that concerns all item kinds